### ISSUE TITLE
A bug in examples/wav2vec/unsupervised/scripts/prepare_audio_v2.sh

### ISSUE THREAD

**Issue by joezhoujinjing** [facebookresearch/fairseq#4615]
## 🐛 Bug

After line 65, we forgot to call  `fairseq/examples/hubert/simple_kmeans/learn_kmeans.py`.

### To Reproduce

Steps to reproduce the behavior (**always include the command you ran**):

1. Run cmd '....'
2. See error

### Environment

 - fairseq Version (e.g., 1.0 or main):
 - PyTorch Version (e.g., 1.0)
 - OS (e.g., Linux):
 - How you installed fairseq (`pip`, source):
 - Build command you used (if compiling from source):
 - Python version:
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:

**Comment by xiabingquan** [comment]
Exactly, otherwise `dump_km_label.py` throws an error. Thanks a lot.
